package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@kazoohr/typeorm-pagination",
-  "repository": "github:KazooHR/typeorm-pagination",
+  "name": "@WorkTangohr/typeorm-pagination",
+  "repository": "github:WorkTangoHR/typeorm-pagination",
```

Comment:
Is this right? Do we have a separate GH account for WorkTango yet?

Reply:
No, this is me just being hyper destructive with my find and replace

Reply:
Specifically I guess my regex filter to only markdown files when I did the find/replace didn't work 🙃